src/foundry/foundry.js/clientDocumentMixin.d.ts

```diff
@@ -17,7 +17,7 @@ type ClientDocumentConstructor<T extends ConstructorOf<foundry.abstract.Document
     new (...args: ConstructorParameters<T>): InstanceType<T> & ClientDocumentMixin<InstanceType<T>>;
   };
 
-declare class ClientDocumentMixin<T extends foundry.abstract.Document<any, any>> {
+export declare class ClientDocumentMixin<T extends foundry.abstract.Document<any, any>> {
```

Comment:
I found no better way to reuse the `ClientDocumentMixin` in the `CanvasDocumentMixin` as `ReturnTypeOf` is not usable with generic functions.

Reply:
yeah, it's fine to export it :+1: